src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/Microsoft.NET.Sdk.Functions.Publish.targets

```diff
@@ -110,6 +110,18 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
           >
   </Target>
 
+  <!--
+    ============================================================
+        _FunctionsPublishDepsCopy
+
+    Copies the deps file to the Functions bin folder
+    ============================================================
+    -->
+  <Target Name="_FunctionsPublishDepsCopy" BeforeTargets="Publish" Condition="'$(SkipFunctionsDepsCopy)' != 'true'">
+    <Exec Command="copy $(PublishDir)$(ProjectName).deps.json $(PublishDir)bin\function.deps.json" />
```

Comment:
nit: Elsewhere we use the Move Task rather than Exec. Probably no difference, but wasn't sure if it had better logging or error handling, etc.

Reply:
Actually -- does this target get used on non-Windows? Will calling `copy` won't work there?